sql/plsql/examples/alter_operator.sql

```diff
@@ -0,0 +1 @@
+ALTER OPERATOR eq_op COMPILE;
```

Comment:
Maybe it makes sense to add more examples on the new syntax. But it's optionally.

Reply:
I only found this one example from the official documentation.